### ISSUE TITLE
ReferenceError: Dependencies is not defined

### ISSUE THREAD

**Issue by fad250** [compodoc/compodoc#681]
##### **Compodoc installed globally or locally ?**
Both

##### **Motivation for or Use Case**

Everytime I reload a document it fails and throw this error. 
##### **Reproduce the error**
1. Run compodoc
2. Update document and hit save
3. Fails on reload and throws the error the following error:

```
 ReferenceError: Dependencies is not defined
 npm ERR! code ELIFECYCLE
 npm ERR! errno 1
```

OS: Windows 10

**Comment by vogloblinsky** [comment]
Fixed if i am not wrong with 1.1.6, please test with last version, thanks.

**Comment by fad250** [comment]
It's works as intended. Thanks!
